### DISCUSSION TITLE
How to disable bell sound?

### DISCUSSION THREAD

**Discussion by a-wing** [neovim/neovim#25232] [Q&A]
### Problem

I use <C+/> hotkey, But, have bell sound

### Expected behavior

Maybe need a config item disable bell sound ?

**Comment by zeertzjq** [comment]
Is `:h 'belloff'` or `:h 'visualbell'` what you are referring to?

**Comment by a-wing** [comment]
I already set:

```vim
set belloff=all
set novisualbell
```

That doesn't work

**Comment by zeertzjq** [comment]
When does the bell sound appear? It doesn't seem to come from Nvim.

**Comment by a-wing** [comment]
Thank you.

Maybe you are right. the bell possibly from MacOS

But, I test
- iTerm2
- Wezterm
- Kitty

This terminal no bell sound.

On the other hand, firefox and chrome get a input focus, <C+/> is bell

Maybe bell from MacOS system, But, Why the `iTerm2`, `Wezterm`, `Kitty` no bell ?

**Comment by glepnir** [comment]
terminal has config disable bell. kitty has `enable_audio_bell no`
